consul/Module.md

```diff
@@ -58,90 +58,91 @@ consul:Client consulClient = new(consulConfig);
 
 Register services in Consul with the given `jsonPayload`.
 ```ballerina
+json jsonPayload = {"ID":"redis", "Name":"redis1", "Address":"localhost", "port":8000, "EnableTagOverride":false};
 var serviceRegister = consulClient->registerService(jsonPayload);
 if (serviceRegister is boolean) {
-   io:println(serviceRegister);
+   io:println("Status of the service: ", serviceRegister);
 } else {
-   io:println(<string>serviceRegister.detail().message);
+   io:println("Error: ", <string>serviceRegister.detail().message);
 }
 ```
 
 Get the details of the service with the given `serviceName`.
 ```ballerina
 var serviceDetails = consulClient->getService(serviceName);
 if (serviceDetails is consul:CatalogService[]) {
-   io:println(serviceDetails);
+   io:println("Details of a particular service: ", serviceDetails);
```

Comment:
```suggestion
   io:println("Details of the service: ", serviceDetails);
```

Why does `getService` return an array though? Isn't it details of a single service?

Reply:
As per the endpoint [1] and API doc [2], it is listing Nodes for Service as an array. 

[1] https://github.com/Biruntha/module-consul/blob/2e49ee51a6df6f188459849492b313307c9f4bbf/consul/consul_endpoint.bal#L80
[2] https://www.consul.io/api/catalog.html#list-nodes-for-service